index.html

```diff
@@ -0,0 +1,83 @@
+                <label for="filter">select category</label>
+                <select id="filter">
+                    <option value="All" selected>All</option>
+                    <option value="tech">Technology</option>
+                    <option value="sports">Sports</option>
+                </select>
+            </div>
+            <div id="main-title">All News</div>
+            <div class="d-flex col" id="subscribe-form">
+                <label for="text-subscribe">Subscribe</label>
+                <form action="#!" class="d-flex row">
+                    <input type="email" id="text-subscribe" placeholder="Email Address">
+                    <button type="submit" id="btn-subscribe">Subscribe</button>
+                </form>
+            </div>
+        </section>
+        
+        <section class="d-flex col">
+            <div class="article-container d-flex row broder-bottom">
+                <div class="article-image"></div>
+                <div class="article-body">
+                    <p class="article-title">Title One</p>
+                    <small>Posted on <b>03 Dec, 2021 </b>// Category: <b>Category One</b></small>
+                    <p class="article-desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos a tempora numquam qui mollitia facilis incidunt necessitatibus eius, aliquam quam aliquid sequi nesciunt error ad sunt esse! Laboriosam, dolore debitis.</p>
+                    <button>Continue Reading</button>
+                </div>
+            </div>
+
+            <div class="article-container d-flex row broder-bottom">
+                <div class="article-image"></div>
+                <div class="article-body">
+                    <p class="article-title">Title One</p>
+                    <small>Posted on <b>03 Dec, 2021 </b>// Category: <b>Category One</b></small>
+                    <p class="article-desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos a tempora numquam qui mollitia facilis incidunt necessitatibus eius, aliquam quam aliquid sequi nesciunt error ad sunt esse! Laboriosam, dolore debitis.</p>
+                    <button>Continue Reading</button>
+                </div>
+            </div>
+
+            <div class="article-container d-flex row">
+                <div class="article-image"></div>
+                <div class="article-body">
+                    <p class="article-title">Title One</p>
+                    <small>Posted on <b>03 Dec, 2021 </b>// Category: <b>Category One</b></small>
+                    <p class="article-desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos a tempora numquam qui mollitia facilis incidunt necessitatibus eius, aliquam quam aliquid sequi nesciunt error ad sunt esse! Laboriosam, dolore debitis.</p>
+                    <button>Continue Reading</button>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <footer class="d-flex col center">@NewsFeed 2021</footer>
```

Comment:
if the header is inside your main tag then why footer is outside?

Reply:
Header tag in outside of the main